apps/go-mod-validator/src/index.ts

```diff
@@ -0,0 +1,79 @@
+import { isGoModReferencingDefaultBranch } from "./github";
+import { getAllGoModDeps } from "./deps";
+import { FIXING_ERRORS } from "./strings";
+import * as github from "@actions/github";
+import * as core from "@actions/core";
+
+const orgPrefix = "github.com/smartcontractkit";
+
+function getContext() {
+  const goModDir: string = core.getInput("go_mod_dir") || process.cwd();
```

Comment:
This being defaulted, even though the `action.yml` defaults it to something else.